velox/row/UnsafeRowFast.cpp

```diff
@@ -27,12 +27,16 @@ int32_t alignBits(int32_t numBits) {
 int32_t alignBytes(int32_t numBytes) {
   return bits::roundUp(numBytes, 8);
 }
+
+bool isFixedWidth(const TypePtr& type) {
```

Comment:
Nice. Let's add the same to UnsafeRowDeserializers.h to reduce copy-paste there as well.